processmanager/processinfo.go

```diff
@@ -88,11 +100,26 @@ func (pm *ProcessManager) updatePidInformation(pid libpf.PID, m *Mapping) (bool,
 		if name, err := os.ReadFile(fmt.Sprintf("/prod/%d/comm", pid)); err == nil {
 			processName = string(name)
 		}
+
+		envVarMap := make(map[string]string)
+		if envVars, err := os.ReadFile(fmt.Sprintf("/proc/%d/environ", pid)); err == nil {
+			splittedVars := strings.Split(string(envVars), "\000")
+			fmt.Println("EnvVars for PID" + strconv.Itoa(int(pid)))
+			for _, envVar := range splittedVars {
+				keyValuePair := strings.Split(envVar, "=")
```

Comment:
This is not correct as values can themselves contain "=".

```suggestion
				keyValuePair := strings.SplitN(envVar, "=", 2)
```